packages/design-picker/src/components/theme-preview/index.tsx

```diff
@@ -47,7 +49,7 @@ const ThemePreview: React.FC< ThemePreviewProps > = ( {
 	const [ isFullyLoaded, setIsFullyLoaded ] = useState( ! isUrlWpcomApi( url ) );
 	const [ viewport, setViewport ] = useState< Viewport >();
 	const [ containerResizeListener, { width: containerWidth } ] = useResizeObserver();
-	const calypso_token = useMemo( () => uuid(), [] );
+	const calypso_token = useMemo( () => iframeToken || uuid(), [ iframeToken ] );
```

Comment:
Is ` || uuid()` still necessary here? As we always pass non-null/undefined value as the `iframeToken` now.

Reply:
The Assembler also uses this component. Since it doesn't need to support SSR, the Assembler doesn't need to worry about passing an iframe token to `ThemePreview`. So `uuid()` is useful in this case!

Reply:
My god, I was searching for `<ThemeWebPreview` usage in the codebase, turned out it is imported as `ThemeWebPreview` only in this file 🤦